openlibrary/core/edits.py

```diff
@@ -54,13 +51,16 @@ def get_requests(cls, limit: int = 50, page: int = 1, mode: str = 'all', order:
         wheres = []
         if kwargs.get("status"):
             wheres.append("status=$status")
-        if "reviewer" in kwargs:
-            wheres.append("reviewer='$reviewer'")
+        if kwargs.get('reviewer') is not None:
+            wheres.append(
+                "reviewer IS NULL" if not kwargs.get('reviewer')
+                else "reviewer=$reviewer"
+            )
         if "submitter" in kwargs:
-            if kwargs.get("submitter") is None:
-                wheres.append("submitter IS NOT NULL")
-            else:
-                wheres.append("submitter=$submitter")
+            wheres.append(
+                "submitter IS NOT NULL" if kwargs.get("submitter") is None
```

Comment:
```suggestion
                # Show all submitters by default, otherwise allow user-specified submitter
                "submitter IS NOT NULL" if kwargs.get("submitter") is None
```